demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -935,6 +937,29 @@ def is_valid_parameters_display_name(self) -> bool:
 
         return True
 
+    def is_valid_integration_file_path(self) -> bool:
+        absolute_file_path = self.file_path
+        integrations_folder = os.path.dirname(absolute_file_path).split('/')[-1]
+        integration_file = os.path.basename(absolute_file_path)
+
+        integration_file, _ = os.path.splitext(integration_file)
+
+        if integrations_folder == 'Integrations':
+            if integration_file.startswith('integration-'):
```

Comment:
I think your logic is off here,  you want to check if this is not occurring and return False in that case.

Reply:
done